base_user_role/wizards/group_groups_into_role.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="group_groups_into_role_wiz_view" model="ir.ui.view">
+        <field name="name">wizard.groups.into.role.wiz.view</field>
+        <field name="model">wizard.groups.into.role</field>
+        <field name="arch" type="xml">
+            <form>
+                <group>
+                    <field name="name" />
+                </group>
+                <footer>
+                    <button
+                        string="Create"
+                        type="object"
+                        class="oe_highlight"
+                        name="create_role"
+                    />
+                    <button string="Cancel" class="oe_link" special="cancel" />
+                </footer>
+            </form>
+        </field>
+    </record>
+    <record id="server_action_group_groups_into_role" model="ir.actions.server">
+        <field name="name">Create role</field>
+        <field name="model_id" ref="model_res_groups" />
+        <field name="binding_model_id" ref="model_res_groups" />
+        <field name="state">code</field>
+        <field name="code">
+            action = {
+                'name': 'Specify a name for your new role',
+                'type': 'ir.actions.act_window',
+                'res_model': 'wizard.groups.into.role',
+                'target': 'new',
+                'views':[(env.ref('base_user_role.group_groups_into_role_wiz_view').id, 'form')],
+                'context': env.context
+            }
+        </field>
+    </record>
```

Comment:
I don't see the interest to use an ir.actions.server. Usually odoo implements ir.actions.act_window. something like : 


    <record id="action_wizard_groups_into_role" model="ir.actions.act_window">
        <field name="name">Create Role<field>
        <field name="type">ir.actions.act_window</field>
        <field name="res_model">wizard.groups.into.role</field>
        <field name="view_type">form</field>
        <field name="view_mode">form</field>
        <field name="target">new</field>
        <field name="binding_model_id" ref="base.model_res_groups" />
    </record>

That way, no need to specify views. And it avoid to have untested python code in server action.  IMO Server action should be used only when we want to execute it via UI that is not the case here.